src/main/java/io/jenkins/plugins/gitlabbranchsource/credentials/PersonalAccessToken.java

```diff
@@ -0,0 +1,17 @@
+package io.jenkins.plugins.gitlabbranchsource.credentials;
+
+import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.util.Secret;
+
+public interface PersonalAccessToken extends StandardCredentials {
+
+    /**
+     * Returns the token.
```

Comment:
This is a super nit, but could we be consistent with spacing between the description and @return? most places have a blank line, this one does not.